napari_builtins/builtins.yaml

```
@@ -132,29 +132,27 @@ contributions:
       accepts_directories: true
       filename_patterns:
         [
-          "*.3fr", "*.arw", "*.avi", "*.bay", "*.bmp", "*.bmq", "*.bsdf",
-          "*.bufr", "*.bw", "*.cap", "*.cine", "*.cr2", "*.crw", "*.cs1",
-          "*.csv", "*.ct", "*.cur", "*.cut", "*.dc2", "*.dcm", "*.dcr", "*.dcx",
-          "*.dds", "*.dicom", "*.dng", "*.drf", "*.dsc", "*.ecw", "*.emf",
-          "*.eps", "*.erf", "*.exr", "*.fff", "*.fit", "*.fits", "*.flc",
-          "*.fli", "*.fpx", "*.ftc", "*.fts", "*.ftu", "*.fz", "*.g3", "*.gbr",
-          "*.gdcm", "*.gif", "*.gipl", "*.grib", "*.h5", "*.hdf", "*.hdf5",
-          "*.hdp", "*.hdr", "*.ia", "*.icns", "*.ico", "*.iff", "*.iim",
-          "*.iiq", "*.im", "*.img.gz", "*.img", "*.ipl", "*.j2c", "*.j2k",
-          "*.jfif", "*.jif", "*.jng", "*.jp2", "*.jpc", "*.jpe", "*.jpeg",
-          "*.jpf", "*.jpg", "*.jpx", "*.jxr", "*.k25", "*.kc2", "*.kdc",
-          "*.koa", "*.lbm", "*.lfp", "*.lfr", "*.lsm", "*.mdc", "*.mef",
-          "*.mgh", "*.mha", "*.mhd", "*.mic", "*.mkv", "*.mnc", "*.mnc2",
-          "*.mos", "*.mov", "*.mp4", "*.mpeg", "*.mpg", "*.mpo", "*.mri",
-          "*.mrw", "*.msp", "*.nef", "*.nhdr", "*.nia", "*.nii.gz", "*.nii",
-          "*.npy", "*.npz", "*.nrrd", "*.nrw", "*.orf", "*.pbm", "*.pcd",
-          "*.pct", "*.pcx", "*.pef", "*.pfm", "*.pgm", "*.pic", "*.pict",
-          "*.png", "*.ppm", "*.ps", "*.psd", "*.ptx", "*.pxn", "*.pxr", "*.qtk",
-          "*.raf", "*.ras", "*.raw", "*.rdc", "*.rgb", "*.rgba", "*.rw2",
-          "*.rwl", "*.rwz", "*.sgi", "*.spe", "*.sr2", "*.srf", "*.srw",
-          "*.sti", "*.stk", "*.swf", "*.targa", "*.tga", "*.tif", "*.tiff",
-          "*.vtk", "*.wap", "*.wbm", "*.wbmp", "*.wdp", "*.webm", "*.webp",
-          "*.wmf", "*.wmv", "*.xbm", "*.xpm", "*.zarr",
+          '*.3fr', '*.arw', '*.avs', '*.bay', '*.bif', '*.bmp', '*.bmq', '*.bsdf',
+          '*.btf', '*.bufr', '*.bw', '*.cap', '*.cine', '*.cr2', '*.crw', '*.cs1',
+          '*.ct', '*.cur', '*.cut', '*.dc2', '*.dcm', '*.dcr', '*.dcx', '*.dds',
+          '*.dicom', '*.dng', '*.drf', '*.dsc', '*.ecw', '*.eer', '*.emf', '*.eps',
+          '*.erf', '*.exr', '*.fff', '*.fit', '*.fits', '*.flc', '*.fli', '*.fpx',
+          '*.ftc', '*.fts', '*.ftu', '*.fz', '*.g3', '*.gbr', '*.gdcm', '*.gel', '*.gif',
+          '*.gipl', '*.grib', '*.h5', '*.hdf', '*.hdf5', '*.hdp', '*.hdr', '*.ia',
+          '*.icns', '*.ico', '*.iff', '*.iim', '*.iiq', '*.im', '*.img', '*.img.gz',
+          '*.ipl', '*.j2c', '*.j2k', '*.jfif', '*.jif', '*.jng', '*.jp2', '*.jpc',
+          '*.jpe', '*.jpeg', '*.jpf', '*.jpg', '*.jpx', '*.jxr', '*.k25', '*.kc2',
+          '*.kdc', '*.koa', '*.lbm', '*.lfp', '*.lfr', '*.lsm', '*.mdc', '*.mef',
+          '*.mgh', '*.mha', '*.mhd', '*.mic', '*.mnc', '*.mnc2', '*.mos', '*.mpo',
+          '*.mri', '*.mrw', '*.msp', '*.ndpi', '*.nef', '*.nhdr', '*.nia', '*.nii',
+          '*.nii.gz', '*.npz', '*.nrrd', '*.nrw', '*.ome.tif', '*.orf', '*.pbm', '*.pcd',
```

```suggestion
          '*.nii.gz', '*.npy', '*.npz', '*.nrrd', '*.nrw', '*.ome.tif', '*.orf', '*.pbm', '*.pcd',
```
Need to have `npy` too!